hangman.py

```diff
@@ -23,7 +23,7 @@
 
 """
 
-stored_letters = list("java")  # converts string to list
+# stored_letters = list("java")  # converts string to list
 guessed_word = len(stored_letters) * ["_"]
 used_letters = list()
 
```

Comment:
A way to do this in one line is using the same case on both sides:
```
stored_letters[i].upper() == guess.upper()
```

Reply:
changes are done